src/JustSaying/Messaging/Middleware/Metrics/StopwatchMiddleware.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.Messaging.Middleware.Handle;
+using JustSaying.Messaging.Monitoring;
+
+namespace JustSaying.Messaging.Middleware.Metrics
+{
+    /// <summary>
+    /// This middleware measures handlers' execution duration and reports the results to <see cref="IMessageMonitor"/>.
```

Comment:
```suggestion
    /// This middleware measures the handler's execution duration and reports the results to <see cref="IMessageMonitor"/>.
```
(this is measuring 1 handler, not multiple right?)

Reply:
Yep, think yours is clearer 👍